eultheme/managers.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+from django.db import models
+from django.db.models import Q
+from django.conf import settings
+from django.utils.timezone import utc
+
+class BannerQuerySet(models.query.QuerySet):
+
+    def active(self):
+        return self.filter(disabled=False, period__enabled=True)
+
+    def first(self):
+        if self.count():
+            return self[0]
+
+        return self
+
+    def deployed(self):
+        if getattr(settings, 'USE_TZ', False):
+            now = datetime.datetime.utcnow().replace(tzinfo=utc)
+        else:
+            now = datetime.datetime.now()
+        query_show = Q(show_on_date__lte=now) & (Q(period__end_time__gte=now) | Q(period__end_time=None))
```

Comment:
Are you sure this is combining the way you want it to? And even if it is, I wonder if it would be more readable to use parentheses and/or line breaks to make it more obvious when reading the code what we are filtering on.

Also, I wonder if would be better to use isnull instead of =None - https://docs.djangoproject.com/en/1.8/ref/models/querysets/#std:fieldlookup-isnull